### PR TITLE
Proposed feature request template

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
Part of https://github.com/narwhals-dev/narwhals/issues/218

Feedback welcome.

### CODE REVIEW

**Comment by MarcoGorelli** [comment]
thanks - can we them what their use case is too please? in particular, if they're looking at adopting narwhals in an open source project, then could they link to the project?

rest looks good to me!

**Comment by DeaMariaLeon** [comment]
I hope it's not too wordy 😬

**Review by MarcoGorelli — APPROVED** [review]
should be good thanks @DeaMariaLeon !
